lib/warnings.js

```diff
@@ -0,0 +1,20 @@
+var chalk = require('chalk')
+
+module.exports = {
+  deprecation: function (template, name) {
+    var initCommand = 'vue init ' + template.replace('-2.0', '') + ' ' + name
+
+    console.log(chalk.red('  This template is deprecated.'))
+    console.log()
+    console.log(chalk.yellow('  Please use this command instead: ') + chalk.green(initCommand))
+    console.log()
+  },
+  vue2Version: function (template, name) {
```

Comment:
While we're here, maybe rename this to a more descriptive `v2BranchIsNowDefault`? What do you think?